app/src/main/kotlin/com/wire/android/ui/authentication/devices/remove/RemoveDeviceItem.kt

```diff
@@ -35,12 +35,12 @@ import com.wire.android.ui.theme.wireTypography
 import com.wire.android.util.formatMediumDateTime
 
 @Composable
-fun RemoveDeviceItem(device: Device, placeholder: Boolean, onRemoveDeviceClick: (Device) -> Unit) {
+fun RemoveDeviceItem(device: Device, placeholder: Boolean, onRemoveDeviceClick: ((Device) -> Unit?)?) {
```

Comment:
i guess it could be `onRemoveDeviceClick: ((Device) -> Unit)?` not `onRemoveDeviceClick: ((Device) -> Unit?)?`